core/new-gui/src/app/workspace/service/workflow-graph/model/mock-workflow-data.ts

```diff
@@ -59,8 +59,8 @@ export const mockMultiInputOutputPredicate: OperatorPredicate = {
   operatorType: 'MultiInputOutput',
   operatorProperties: {
   },
-  inputPorts: ['input-0','input-1','input-2'],
-  outputPorts: ['output-0','output-1','output-2'],
+  inputPorts: ['input-0', 'input-1', 'input-2'],
```

Comment:
revert this unnecessary change

Reply:
This space is required by TSLint.
When I save this file, the VS Code will add this space automatically.

Reply:
Actually when I save this file, the TSLint change the space automatically.